sansa-ml-flink/src/main/scala/net/sansa_stack/ml/flink/clustering/RDFByModularityClustering.scala

```diff
@@ -142,10 +142,11 @@ object RDFByModularityClustering {
     val ((bestList1, bestList2), bestQ: Double): ((List[String], List[String]), Double) = {
 
       val s = deltaQDataSet.groupBy(0).sortGroup(1, Order.ASCENDING).reduceGroup {
-        (in, out: Collector[((List[String], List[String]), Double)]) =>
-          var prev: (String, String) = null
+        (in: Iterator[((List[String], List[String]), Double)], out: Collector[((List[String], List[String]), Double)]) =>
+          var prev: ((List[String], List[String]), Double) = null
           for (t <- in) {
             if (prev == null || prev != t) {
+              prev = t
```

Comment:
IMHO I can completely agree to name variables this short to keep code dense... but I would prefer at least having a  comment describing in few words the need/functionality of variables